code/modules/paperwork/silicon_photography.dm

```diff
@@ -151,7 +151,7 @@
 	deletepicture(src)
 
 /obj/item/camera/siliconcam/proc/getsource()
-	if(istype(src.loc, /mob/living/silicon/ai))
+	if(isAI(src.loc))
```

Comment:
nit but this could be `loc` instead of `src.loc`, claim out of scope if you'd like